Assets/Scripts/Quest System/QuestManager.cs

```diff
@@ -251,5 +264,26 @@ public void AddReward(Quest quest)
         }
     }
 
+    public bool CheckSubquest(Quest quest)
+    {
+        if (quest.IsSubQuest == true)
+            return true;
+        return false;
+    }
```

Comment:
A bit simpler, plus the name of the function should start with 'Is'
```suggestion
    public bool IsSubquest(Quest quest)
    {
        return quest.isSubQuest;
    }
```